frontend/javascripts/oxalis/model/sagas/quick_select_ml_saga.ts

```diff
@@ -76,8 +77,19 @@ function* getMask(
     layerName,
     mag,
     maskBoxMag1,
-    userBoxRelativeToMaskInMag.getMinUV(activeViewport),
-    userBoxRelativeToMaskInMag.getMaxUV(activeViewport),
+    usePointPrompt
+      ? {
+          type: "POINT",
+          pointX: userBoxRelativeToMaskInMag.getMinUV(activeViewport)[0],
+          pointY: userBoxRelativeToMaskInMag.getMinUV(activeViewport)[1],
+        }
+      : {
+          type: "BOUNDING_BOX",
+          selectionTopLeftX: userBoxRelativeToMaskInMag.getMinUV(activeViewport)[0],
+          selectionTopLeftY: userBoxRelativeToMaskInMag.getMinUV(activeViewport)[1],
+          selectionBottomRightX: userBoxRelativeToMaskInMag.getMaxUV(activeViewport)[0],
+          selectionBottomRightY: userBoxRelativeToMaskInMag.getMaxUV(activeViewport)[1],
+        },
```

Comment:
Hmm maybe "extract" the calls to `userBoxRelativeToMaskInMax/Min` to have less duplication?

Kinda like:
```
const topLeft = userBoxRelativeToMaskInMag.getMinUV(activeViewport);
const bottomRight = userBoxRelativeToMaskInMag.getMaxUV(activeViewport);
```

And then reuse them in the yield call
```suggestion
    usePointPrompt
      ? {
          type: "POINT",
          pointX: topLeft[0],
          pointY: topLeft[1],
        }
      : {
          type: "BOUNDING_BOX",
          selectionTopLeftX: topLeft[0],
          selectionTopLeftY: topLeft[1],
          selectionBottomRightX: bottomRight[0],
          selectionBottomRightY: bottomRight[1],
        },
```